src/services/access/useRevokeUserToken.ts

```diff
@@ -15,15 +15,15 @@ export function useRevokeUserToken({ provider }) {
         }
       }
     `
-      const variables = { input: { tokenid } }
-
       return Api.graphqlMutation({
         provider,
         query,
-        variables,
+        variables: {
+          input: { tokenid },
+        },
         mutationPath: 'revokeUserToken',
       }).then((res) => {
-        queryClient.invalidateQueries('sessions')
+        queryClient.invalidateQueries(['sessions'])
```

Comment:
jw, was this needed?

Reply:
(it probably didn't work fully properly before 😬)